src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -454,6 +470,250 @@ components:
         - actionedBy
         - timestamp
         - correlationId
+    CiagInductionResponse:
+      title: CiagInductionResponse
+      type: object
+      description: A Prisoner's CIAG Induction containing information, such as their education history and future work interests.
+      properties:
+        reference:
```

Comment:
A new field which I'm assuming won't break the UI.

Reply:
Presumably this is in place of the current `id` field?; and presumably we'll need to get the CIAG UI to read this field (`reference`) rather than `id` and to use it in the URI path when making the subsequent `PUT` request ?

Reply:
So we'll need to remember that as part of our migration plans, as and when we switch the UI over to using this new API ?

Reply:
It's a new field. The `offenderId` is the PK for the induction!